src/app/workspace/[workspaceId]/calendar/page.tsx

```
@@ -96,9 +96,13 @@ const CalendarPage = () => {
 	const [currentDate, setCurrentDate] = useState(new Date());
 	const workspaceId = useWorkspaceId();
 
+	if (!workspaceId) {
+		return null;
+	}
+
 	// Track user activity and time spent on calendar page
 	useTrackActivity({
```

Biome (2.4.15)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)